tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -122,8 +122,9 @@ Standard-Section: "ASTs" TopLevelStat*
                   MATCHtpt       Length bound_Term? sel_Term CaseDef*              -- sel match { CaseDef } where `bound` is optional upper bound of all rhs
                   BYNAMEtpt             underlying_Term                            -- => underlying
                   SHAREDterm            term_ASTRef                                -- Link to previously serialized term
-                  HOLE           Length idx_Nat tpe_Type arg_Tree*                 -- Splice hole with index `idx`, the type of the hole `tpe`, type and term arguments of the hole `arg`s
-
+                  HOLE           Length idx_Nat tpe_Type (HOLETYPES Length targ_Type*)? arg_Tree*
+                                                                                   -- Splice hole with index `idx`, the type of the hole `tpe`, type arguments of the hole `targ`s and term arguments of the hole `arg`s
+                                                                                      -- Note: From 3.0 to 3.3 `args` could contain type arguments as well. This is no longer the case.
```

Comment:
To be clear, args when you say "type arguments" here you mean TypeTrees, as opposed to the use of "type arguments" in the previous sentence where they are Types?